dev/com.ibm.websphere.appserver.features/visibility/public/messagingServer-2.0/resources/l10n/io.openliberty.messagingServer-2.0.properties

```diff
@@ -0,0 +1,18 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables an embedded messaging server that is Jakarta Messaging \
+ compliant. Applications can send and receive messages using the messagingClient feature.
```

Comment:
**change**
Applications can send and receive messages using the messagingClient feature.
**to**
Applications can send and receive messages by using the messagingClient feature.

Reply:
Change made.